### ISSUE TITLE
Create generator commands for FW/1

### ISSUE THREAD

**Issue by bdw429s** [framework-one/fw1#356]
If someone makes a set of FW/1 commands, they can be placed on Forgebox and anyone can install them via `install`.  Ex:
```CommandBox> install fw1-commands```
```CommandBox> fw1 create app```
```CommandBox> fw1 create controller```

Here are the docs for creating commands:
 http://ortus.gitbooks.io/commandbox-documentation/content/developing_commands/developing_commands.html

Here are the APi docs for the current ColdBox generator commands for examples:
http://apidocs.ortussolutions.com/commandbox/2.0.0/index.html?commandbox/commands/coldbox/create/package-summary.html

And here is the code for reference:
https://github.com/Ortus-Solutions/commandbox/tree/master/src/cfml/commands/coldbox/create

**Comment by cfchef** [comment]
I've started to toy with this, as a package of commands on ForgeBox, tying to a repo I created ([fw1-commands](https://github.com/cfchef/fw1-commands)) that only generates some skeleton apps (for now). The innards are based off of what I saw being done with the ColdBox commands which is all pretty slick and a good primer since I'm new to how CommandBox works. With that said, some of it might be rough around the edges ;)

I want to see about adding a few more features and getting feedback on what a specific skeleton should contain when being generated but otherwise I think it's a step in the right direction.
